sql/core/src/main/scala/org/apache/spark/sql/execution/SparkPlanInfo.scala

```diff
@@ -47,7 +47,7 @@ class SparkPlanInfo(
   }
 }
 
-private[sql] object SparkPlanInfo {
+private[execution] object SparkPlanInfo {
```

Comment:
If it is something internal to the implementation (i.e. not part of some public field for expressions or commands or query plan), I'm keeping them private for tighten visibility.